tools/Makefile.unix

```diff
@@ -307,17 +307,24 @@ $(ARCH_SRC)/chip: .clean_context
 ifneq ($(CONFIG_ARCH_CHIP),)
```

Comment:
change to:
ifeq ($(ARCH_CHIP_CUSTOM),y)
else ifneq($(CONFIG_ARCH_CHIP),)
endif